source/jsonnet/england-wales/individual/blocks/employment/mainly_work_in_uk.jsonnet

```diff
@@ -91,6 +91,22 @@ local question(isProxy) = {
         ],
       },
     },
+    {
+      goto: {
+        block: 'workplace-address',
+        when: [
+          {
+            id: 'workplace-type-answer',
+            condition: 'not set',
+          },
+          {
+            id: 'mainly-work-in-uk-answer',
+            condition: 'equals',
+            value: 'Yes',
```

Comment:
The other rules use:
```
condition: 'not equals',
value: 'No',
```

Which is almost the same, but also accounts for `mainly-work-in-uk-answer` not being answered. Should this be the same?

Reply:
I've just realised there is also a rule for both answers not being set, that rule could be updated instead of adding new rule (and it would also be consistent with the other rules).